lib/frontend/app.js

```diff
@@ -144,7 +145,6 @@ module.exports = (appTokenGenerator) => {
     })
   }
 
-  app.use(sentryMiddleware)
```

Comment:
Moved this to before `Sentry.Handlers.errorHandler()`. If it comes after, it isn't guaranteed to have run by the time the error is sent to Sentry.